src/libAtomVM/bitstring.h

```diff
@@ -90,7 +90,13 @@ extern "C" {
 enum BitstringFlags
 {
     LittleEndianInteger = 0x2,
-    SignedInteger = 0x4
+    SignedInteger = 0x4,
+    NativeEndianInteger = 0x10,
+#ifdef __ORDER_LITTLE_ENDIAN__
+    LittleEndianIntegerMask = LittleEndianInteger | NativeEndianInteger
+#else
+    LittleEndianIntegerMask = LittleEndianInteger
```

Comment:
Just for my knowledge, does any standard C version allow this?

Reply:
I am confused. Are you referring to having a constant expression in the enumerator?
It is in C89 §3.5.2.2, C90 §6.5.2.2, C99, §6.7.2.2 and C11, §6.7.2.2.

Reply:
> I am confused. Are you referring to having a constant expression in the enumerator?
It is in C89 §3.5.2.2, C90 §6.5.2.2, C99, §6.7.2.2 and C11, §6.7.2.2.

yes, the question was that. Thanks for the answer, TIL that it is not a C++ feature.